x-pack/plugins/threat_intelligence/public/modules/indicators/lib/field_value.ts

```diff
@@ -29,3 +30,12 @@ export const getIndicatorFieldAndValue = (
     value,
   };
 };
+
+/**
+ * Checks if field and value are correct
+ * @param field Indicator string field
+ * @param value Indicator string|null value for the field
+ * @returns true if correct, false if not
+ */
+export const fieldAndValueValid = (field: string | null, value: string | null): boolean =>
+  value != null && value !== '' && value !== EMPTY_VALUE && field != null && field !== '';
```

Comment:
any partucular reason to do `value != null && value !== ''` instead of `!value`. Also not sure why non-strict comparison with null. I'd vote for a simpler version

Reply:
ok I was confused for a sec... `!value` will not work with empty strings, but `!!value will`. It's cleaned up now! :)